public/js/components/App.js

```diff
@@ -29,7 +29,7 @@ const App = React.createClass({
 
   getInitialState() {
     return {
-      searchOn: false
+      searchOn: true
```

Comment:
Not sure if I love it being on by default. It adds more UI and the need to explain it anyway. Why not just have the explanation, since we'll need it anyway? The search bar is basically a modal and it feels weird to open the debugger by default with a modal.

For these kinds of changes we should loop in Helen also.